app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
     pass
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = []
+    for person_data in people:
+        person = Person(person_data["name"], person_data["age"])
+
+        if "wife" in person_data and person_data["wife"]:
+            person.wife = Person.people.get(person_data["wife"])
```

Comment:
When assigning `person.wife`, ensure that the `wife` exists in the `Person.people` dictionary. If not, this will result in `None`, which could lead to issues if not handled properly.